apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Stores a pattern to identify a keyword.
      */
-    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][a-zA-Z0-9]*)+";
+    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][_a-zA-Z0-9]*)+";
```

Comment:
please evaluate the pattern with
myVar_
_myVar
_myVar_001
myVar_001
_my_Var
_my_Var_
_my_Var_001